dbexporter/docker-compose-fem-dbexporter-mongodb.yml

```diff
@@ -7,8 +7,8 @@ services:
   fem_mongodb-exporter:
     image: percona/mongodb_exporter:2.37.0
     restart: unless-stopped
-    container_name: fem_mongodb_exporter
-    hostname: fem_mongodb_exporter
+    container_name: fem-mongodb_exporter
```

Comment:
Since it is part of the container, I suppose the other "_" should also be changed. Do you agree?